docs/operate/get-started/supported-hardware/_index.md

```diff
@@ -82,9 +84,9 @@ Modules in the list above are officially supported and maintained by Viam if and
 
 ### For use with ESP-32 microcontrollers
 
-The following is a selection of components (some built-ins and some modules) written for use with `viam-micro-server`.
+The following is a selection of components (some built-ins and some modules) written for use with the Micro-RDK.
 To use any of the built-in components, configure them according to their readmes.
-To use a module with `viam-micro-server`, you can either install the pre-built `viam-micro-server` firmware that ships with a few common modules, or [build your own firmware that combines `viam-micro-server` with one or more modules](/operate/get-started/other-hardware/micro-module/).
+To use a module with the Micro-RDK, you can either install the pre-built `viam-micro-server` firmware that ships with a few common modules, or [build your own firmware that combines the Micro-RDK with one or more modules](/operate/get-started/other-hardware/micro-module/).
```

Comment:
```suggestion
You can either install the pre-built `viam-micro-server` firmware that ships with a few common modules, or [build your own firmware that combines the Micro-RDK with one or more modules](/operate/get-started/other-hardware/micro-module/).
```

Reply:
```suggestion
You can either install the pre-built `viam-micro-server` firmware that ships with a few common modules, or [build your own firmware that combines the Micro-RDK with one or more modules of your choice](/operate/get-started/other-hardware/micro-module/).
```
Okay! Also realizing maybe this makes more sense

Reply:
LGTM